### ISSUE TITLE
Recommendation 11: The Planetary Data Ecosystem should regularly (on a one- to two-year time scale) assess the Findability, Accessibility, Interoperability, and Reusability (FAIR) ...

### ISSUE THREAD

**Issue by PDE-CS** [PDE-CS/Planetary-Data-Ecosystem#11]
Recommendation 11 
The Planetary Data Ecosystem should regularly (on a one- to two-year
time scale) assess the Findability, Accessibility, Interoperability, and Reusability (FAIR) of data across each PDE element for machine-actionable access to data. This assessment should be used to establish the priorities for Ecosystem management and advisory groups.

**Comment by PDE-CS** [comment]
PDS management response:

- The ongoing development and documentation of PDS4, coupled with the migration of PDS 3 data to PDS 4, development of related tools, registries and data dictionaries directly demonstrates the improving FAIRness of the PDS archives. 
- PDS Data Design Working Group oversees and continues to evolve the PDS4 Information Model. DDWG has the experience and expertise to stand ready to contribute to FAIR-metadata standards across all PDE elements if such effort is initiated by HQ.
- How do we interact with non-PDS repositories that are not as well-documented?
